bots/discord/nodemon.js/README.md

```diff
@@ -0,0 +1,7 @@
+# nodemon.js
```

Comment:
Please [copy the readme](https://github.com/parkervcp/eggs/tree/master/bots/discord/discord.js#readme) from other discord generic eggs that contain instructions on how to configure the start config. It's the most common question and complaint we get.